src/testing/fixtures/algorand-fixture.ts

```diff
@@ -84,21 +84,26 @@ export function algorandFixture(fixtureConfig?: AlgorandFixtureConfig, config?:
   const indexer = fixtureConfig.indexer ?? ClientManager.getIndexerClient(fixtureConfig.indexerConfig!)
   const kmd = fixtureConfig.kmd ?? ClientManager.getKmdClient(fixtureConfig.kmdConfig!)
   let context: AlgorandTestAutomationContext
-  let algorand: AlgorandClient
+  let algorand = AlgorandClient.fromClients({ algod, indexer, kmd })
```

Comment:
If a dev is using this before calling `newScope`, then the effect of detecting localnet and increasing the default validity window will not have run.

I'm wondering if we even need to create a default instance on calling the `algorandFixture` function?